uber/models.py

```diff
@@ -1252,6 +1263,9 @@ class Attendee(MagModel, TakesPaymentMixin):
     ec_phone      = Column(UnicodeText)
     cellphone     = Column(UnicodeText)
 
+    # Represents a request for hotel booking info during preregistration
+    requested_hotel_info = Column(Boolean, default=False)
```

Comment:
If you make the default True, you should probably add verbiage to make it clear that they ***will*** get the hotel link e-mail unless they opt-out. I'd prefer that the staff import opt them in automagically versus forcing all attendees to uncheck the box.